### PR TITLE
Fix preinstalled scylla gce

### PR DESCRIPTION
**Pull request by fgelcer**
## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by soyacz** [comment]
@fgelcer please provide some description

**Comment by fgelcer** [comment]
> @fgelcer please provide some description

re-submitting what was reverted on #6409 with the required fixes

**Comment by soyacz** [comment]
> > @fgelcer please provide some description
> 
> re-submitting what was reverted on #6409 with the required fixes

Would be good to include the info what was wrong and how it was fixed.

**Comment by fgelcer** [comment]
> > > @fgelcer please provide some description
> > 
> > 
> > re-submitting what was reverted on #6409 with the required fixes
> 
> Would be good to include the info what was wrong and how it was fixed.

there is a new commit, that adds what was missing from previous code

**Comment by fgelcer** [comment]
@Mark-Gurevich , what is the status of testing this PR?

**Comment by Mark-Gurevich** [comment]
Running [Artifact tests](https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/mark/job/gce/job/artifacts/) , [Artifacts offline tests](https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/mark/job/gce/job/artifacts-offline-install/), and [rolling upgrade tests](https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/mark/job/gce/job/rolling-upgrade/) in staging according to [comment](https://github.com/scylladb/scylla-cluster-tests/pull/6363#issuecomment-1646813515)

**Comment by fgelcer** [comment]
> Running [Artifact tests](https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/mark/job/gce/job/artifacts/) , [Artifacts offline tests](https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/mark/job/gce/job/artifacts-offline-install/), [rolling upgrade tests](https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/mark/job/gce/job/rolling-upgrade/) and [manager tests](https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/mark/job/gce/job/manager/) staging according to [comment](https://github.com/scylladb/scylla-cluster-tests/pull/6363#issuecomment-1646813515)

based on the runs above, seems everything now is working as expected, hence this PR is ready to be merged

File: jenkins-pipelines/longevity-mv-si-24h-gce.jenkinsfile [hunk 1]
```diff
@@ -1,12 +0,0 @@
-#!groovy
```
Comment (vponomaryov) [inline jenkins-pipelines/longevity-mv-si-24h-gce.jenkinsfile]: Is it intentional file deletion? It doesn't look related to the goal of the PR.

**Review by vponomaryov — CHANGES_REQUESTED** [review]
First commit deletes `jenkins-pipelines/longevity-mv-si-24h-gce.jenkinsfile` file
Second adds it back.
Also, second one has changes that must have been in first one.
The third commit description says `use_preinstalled_scylla: true` , but it must be opposite - `use_preinstalled_scylla: false`.

Please, fix this mess.

File: jenkins-pipelines/longevity-mv-si-24h-gce.jenkinsfile (hunk 1 above)
Comment (Mark-Gurevich) [reply jenkins-pipelines/longevity-mv-si-24h-gce.jenkinsfile]: Squashed all commits to a single commit with clear message description.

**Comment by Mark-Gurevich** [comment]
> > Running [Artifact tests](https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/mark/job/gce/job/artifacts/) , [Artifacts offline tests](https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/mark/job/gce/job/artifacts-offline-install/), [rolling upgrade tests](https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/mark/job/gce/job/rolling-upgrade/) and [manager tests](https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/mark/job/gce/job/manager/) staging according to [comment](https://github.com/scylladb/scylla-cluster-tests/pull/6363#issuecomment-1646813515)
> 
> based on the runs above, seems everything now is working as expected, hence this PR is ready to be merged

Indeed all runs have passed, while those which failed were able to setup Scylla. 
The failures are not related to the current change but to other issues.
The PR is re-based on master and ready to be merged.

**Review by soyacz — APPROVED** [review]
LGTM

**Review by vponomaryov — APPROVED** [review]
LGTM

**Comment by Mark-Gurevich** [comment]
@scylladb/qa-maintainers
